### DISCUSSION TITLE
Error displayed when manually entering Markdown link

### DISCUSSION THREAD

**Discussion by madduck** [requarks/wiki#3370] [Help / Questions]
While editing Markdown, if I start a link by typing `[link text](…`, then as soon as I enter the opening parenthesis, an error message is displayed across the top, and it stays there until I close the link with `)`. Then it just disappears.

![image](https://user-images.githubusercontent.com/195073/81233738-71896c80-904b-11ea-8733-64ae0ec83ee6.png)

**Host Info (please complete the following information):**
 - OS: Debian 10
 - Browser: Firefox 75, clean profile
 - Wiki.js version: 2.3.81
 - Database engine: PostgreSQL 11.7

**Comment by NGPixel** [comment]
Check in the dev tools for failed network requests and post the response here.

**Comment by madduck** [comment]
```
12:01:11.440 [GraphQL error]: Message: Cannot read property 'results' of undefined, Location: [object Object], Path: pages,search app.3db9914c6708a1a6227a.js:2:2424
    N https://wiki.edsfellas.nz/js/app.3db9914c6708a1a6227a.js:2
    N https://wiki.edsfellas.nz/js/app.3db9914c6708a1a6227a.js:2
    next https://wiki.edsfellas.nz/js/app.3db9914c6708a1a6227a.js:2
    b https://wiki.edsfellas.nz/js/app.3db9914c6708a1a6227a.js:60
    y https://wiki.edsfellas.nz/js/app.3db9914c6708a1a6227a.js:60
    value https://wiki.edsfellas.nz/js/app.3db9914c6708a1a6227a.js:60
    next https://wiki.edsfellas.nz/js/app.3db9914c6708a1a6227a.js:2
    next https://wiki.edsfellas.nz/js/app.3db9914c6708a1a6227a.js:2
    next https://wiki.edsfellas.nz/js/app.3db9914c6708a1a6227a.js:2
    b https://wiki.edsfellas.nz/js/app.3db9914c6708a1a6227a.js:60
    y https://wiki.edsfellas.nz/js/app.3db9914c6708a1a6227a.js:60
    value https://wiki.edsfellas.nz/js/app.3db9914c6708a1a6227a.js:60
    batchHandler https://wiki.edsfellas.nz/js/app.3db9914c6708a1a6227a.js:2
```

**Comment by misharigot** [comment]
For me, whenever I type `[](`, as soon as I press `(`, it will be replaced with `[]/tech/ssm-scp)`.
This is a link to a page on my wiki, but for some reason it always only auto completes to that page, and it isn't even valid markdown, as it misses the leading `(`.

This makes it hard to insert links in markdown, as I have to fix the link each time manually.

There are no failed network requests. My graphql network request has the following result:

```
[
   {
      "data":{
         "pages":{
            "search":{
               "results":[
                  {
                     "title":"SCP with SSM",
                     "path":"tech/ssm-scp",
                     "locale":"en",
                     "__typename":"PageSearchResult"
                  }
               ],
               "totalHits":1,
               "__typename":"PageSearchResponse"
            },
            "__typename":"PageQuery"
         }
      }
   }
]
```
